### PR TITLE
CLB: Illithid Harvester // Plant Tadpoles and support

### LINKED ISSUE #253: Illithid Harvester // Plant Tadpoles
SetState needs to have `FaceDownSetType` like `ChangeZoneEffect` has, with that Horror Type can be set

### PR DESCRIPTION
**Pull request by Northmoc**
- closes #253 

@Hanmac your notes in the issue were very helpful - thanks

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/illithid_harvester_plant_tadpoles.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+AlternateMode:Adventure
+Oracle:Ceremorphosis — When Illithid Harvester enters the battlefield, turn any number of target tapped nontoken creatures face down. They're 2/2 Horror creatures.
+
+ALTERNATE
+
+Name:Plant Tadpoles
+ManaCost:X U U
+Types:Sorcery Adventure
+A:SP$ Tap | ValidTgts$ Creature | TgtPrompt$ Select X target creatures | TargetMin$ X | TargetMax$ X | AlwaysRemember$ True | SubAbility$ DBPump | SpellDescription$ Tap X target creatures.
+SVar:DBPump:DB$ Pump | Defined$ Remembered | KW$ HIDDEN This card doesn't untap during your next untap step. | Duration$ Permanent | SubAbility$ DBCleanup | StackDescription$ SpellDescription | SpellDescription$ They don't untap during their controllers' next untap steps.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/illithid_harvester_plant_tadpoles.txt:17]: Why not just use `Defined$ Targeted` here?

**Comment by Northmoc** [comment]
I think this is good to go?

**Comment by tool4ever** [comment]
I couldn't fully check this yet, copy effects aren't my best area

File: forge-game/src/main/java/forge/game/ability/effects/SetStateEffect.java [hunk 2]
```diff
@@ -196,4 +204,34 @@ public void resolve(final SpellAbility sa) {
             game.getAction().reveal(transformedCards, p, true, "Transformed cards in ");
         }
     }
+
+    private static void setFaceDownState(Card c, SpellAbility sa) {
```
Comment (tool4ever) [inline forge-game/src/main/java/forge/game/ability/effects/SetStateEffect.java:208]: Can you just move the one from ChangeZoneEffect to a common place?

Comment (Northmoc) [reply forge-game/src/main/java/forge/game/ability/effects/SetStateEffect.java:208]: Absolutely!

**Review by tool4ever — APPROVED** [review]

**Pull request merged**